Soccar [UI]/src/nl/socnet/message/PlayerMovedMessage.java

```diff
@@ -0,0 +1,45 @@
+package nl.socnet.message;
+
+import nl.soccar.library.enumeration.HandbrakeAction;
+import nl.soccar.library.enumeration.SteerAction;
+import nl.soccar.library.enumeration.ThrottleAction;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+import nl.socnet.message.handler.PlayerMovedMessageHandler;
+
+/**
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.PLAYER_MOVED_MESSAGE_ID, handler = PlayerMovedMessageHandler.class)
+public final class PlayerMovedMessage extends Message {
+
+    private final SteerAction steerAction;
+    private final HandbrakeAction handbrakeAction;
+    private final ThrottleAction throttleAction;
+    
+    public PlayerMovedMessage(SteerAction steerAction, HandbrakeAction handbrakeAction, ThrottleAction throttleAction) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)